grpc/src/main/java/com/linecorp/armeria/server/grpc/AbstractUnframedGrpcService.java

```diff
@@ -138,7 +138,8 @@ protected void frameAndServe(
             RequestHeaders grpcHeaders,
             HttpData content,
             CompletableFuture<HttpResponse> res,
-            @Nullable Function<HttpData, HttpData> responseBodyConverter) {
+            @Nullable Function<HttpData, HttpData> responseBodyConverter,
+            @Nullable MediaType responseContentType) {
```

Comment:
`responseContentType ` isn't nullable anymore?
```suggestion
            MediaType responseContentType) {
```